tests/suite-integration/PluginFlyvemdmAgent.php

```diff
@@ -644,19 +648,22 @@ public function testPurgeEnroledAgent() {
       $this->boolean($mqttUser->getByUser($serial))->isFalse();
       $computer = new \Computer();
       $this->boolean($computer->getFromDB($computerId))->isFalse();
+
+       $this->boolean($user->getFromDb($testUserId))->isTrue();
    }
 
    /**
-    * Test the purge of an agent deletes the user if he no longer has any agent
-    *
+    * Test the purge of an agent the user must persist if he no longer has any agent
     *
+    * @tags thisIsOne
```

Comment:
What is the purpose of this annotation ?

Reply:
atoum's tags is a helper to do only that method when you run the test if you use the -t argument. http://docs.atoum.org/en/latest/running_tests.html#filtres-par-tag